### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by DukeJukemEG** [valinet/ExplorerPatcher#3737] [Ideas]
I'm attempting to use the Full Screen Start Menu with the Windows 10 Skin and would like the Start Menu to open the "All apps" section with the Windows 10 Skin please. You can do it with the Windows 11 Skin which I thought was really awesome. I gotta get that feeling with the Windows 10 Skin. 1 small thing I'd like to see which might not be possible in relation to this is to have the Start Menu appear while keeping the taskbar hidden. I'm attempting to mimic Linux behavior. Maybe a search UI within the W10 menu too.

I ask for this because I'm trying to keep everything out of my sight as much as possible and when I need something I can hit the Windows Key and find everything or if I don't want a massive UI in my face I can do the PowerToys search thing.

I'm trying to navigate to Linux one day and I feel making Windows mimic a Linux-like UI is the best way to enter the ecosystem instead of diving into a distro and watching videos. I'm not worried about installing apps on Linux anymore because that issue is mostly dead. I'm worried about looking at an unfamiliar home and running away so I should make Windows appear more like Linux before I leave.

Amazing software. Thank you for taking the time to do this for us, for nothing. Much love.

**Comment by wuyilingwei** [comment]
#3741

Same idea, I hope win10 can also support full-screen direct entry into the application list instead of the abandoned tiles.
